src/main/config/messages_template.xml

```diff
@@ -62,7 +62,7 @@ See the accompanying LICENSE file for applicable license.
     <response>Please ensure that '%1' exists and that you have permission to access it.</response>
   </message>
   
-  <message id="DOTJ007E" type="ERROR">
+  <message id="DOTJ007W" type="WARN">
```

Comment:
I wasn't 100% sure about changing the number - definitely can't do that in a hotfix but seemed possibly OK in a release. I still think of the `E` or `W` as part of the error code, but that's because I've been used to that convention going back years before DITA-OT. I do think that a lot of our customers are used to keying off of that, and would think that `[DOTJ007E][WARNING]` is a bug. [If we were starting over with a new system, I'd agree though that `DOT007` would be the right number, without duplicating the error level.]